packages/teams-js/src/public/search.ts

```diff
@@ -62,6 +55,12 @@ export namespace search {
    * search (by pressing Enter for example). Should be used to display the full list of search results. 
    * The value of {@link SearchQuery.searchTerm} is the complete query the user entered in the search box.
    *
+   * @param onChangeHandler - This optional handler will be called when the user first starts using the
+   * host's search box and as the user types their query. Can be used to put your application into a 
+   * word-wheeling state or to display suggestions as the user is typing. 
+   * 
+   * This handler will be called with an empty {@link SearchQuery.searchTerm} when search is beginning, and subsequently,
+   * with the current contents of the search box.
    * @example
    * ``` ts
```

Comment:
Can you please update the example and move the code on lines 67-69 to after lines 73-75?